packages/@aws-cdk/aws-events-targets/lib/api-gateway.ts

```diff
@@ -70,11 +70,14 @@ export interface ApiGatewayProps extends TargetBaseProps {
 }
 
 /**
- * Use an API Gateway REST APIs as a target for Amazon EventBridge rules.
+ * Use an API Gateway REST API as a target for Amazon EventBridge rules.
  */
-export class ApiGateway implements events.IRuleTarget {
+export class ApiGatewayTarget implements events.IRuleTarget {
```

Comment:
As far as I can tell, none of the other classes end in `Target`. 

I'm not sure this one should buck the trend.